app/src/main/java/com/android/sample/ui/account/AccountScreen.kt

```diff
@@ -1,11 +1,148 @@
 package com.android.sample.ui.account
 
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import coil.compose.rememberAsyncImagePainter
+import com.android.sample.R
 import com.android.sample.model.user.UserViewModel
-import com.android.sample.ui.EmptyScreen
 import com.android.sample.ui.navigation.NavigationActions
+import com.android.sample.ui.navigation.TopLevelDestinations
+import com.android.sample.ui.utils.PlateSwipeScaffold
+import com.android.sample.ui.utils.RecipeList
 
 @Composable
 fun AccountScreen(navigationActions: NavigationActions, userViewModel: UserViewModel) {
-  EmptyScreen(navigationActions, "Account Screen")
+  userViewModel.getCurrentUser()
+
+  PlateSwipeScaffold(
+      navigationActions = navigationActions,
+      selectedItem = TopLevelDestinations.ACCOUNT.route,
+      content = { padding ->
+        val userName = userViewModel.userName.collectAsState()
+        val profilePictureUrl = userViewModel.profilePictureUrl.collectAsState()
+
+        Column(
+            modifier = Modifier.fillMaxSize().padding(padding).padding(8.dp),
+            verticalArrangement = Arrangement.spacedBy(16.dp, Alignment.Top),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+          Spacer(modifier = Modifier.height(16.dp))
+
+          ProfilePicture(profilePictureUrl.value, modifier = Modifier.weight(.4f))
+
+          Text(
+              text = userName.value ?: stringResource(R.string.account_screen_default_user_name),
```

Comment:
I see that when `userName` has no value, we’re using a default one. However, from my testing, each user already has a default username (an auto-generated UID). This raises a few potential options: (1) we could assign a more meaningful default username for everyone, or (2) prompt each user to set a username upon registration. 

That said, I realize this might be slightly out of scope, as this PR is focused on the account screen UI and not the full underlying logic, if I understand correctly.

Reply:
I agree that we should have a prompt to ask the user to give a username, when first signing in. Although, as you said, this is out of the scope of this PR. 
It's more meant as a failsafe, so that something does appear and their isn't just a blank space.